app/controllers/items_controller.rb

```diff
@@ -0,0 +1,36 @@
+class ItemsController < ApplicationController
+
+  def new
+    @user = current_user
+    @item = Item.new 
+#    authorize @item
+  end
+
+  def create
+    @user = current_user
+    @items = @user.items
+
+    @item = current_user.items.build( items_params )
+    @new_item = Item.new
+    
+#    authorize @item
+    if @item.save
+      flash[:notice] = "New to-do item created."
```

Comment:
For some reason new items are saved and shown on the screen (via the javascript) but no flash appears unless I refresh the whole page.